### ISSUE TITLE
Ranking page out of date

### ISSUE THREAD

**Issue by jwrober** [longturn/www#38]
The ranking page is woefully out of date. Do we still need it?

**Comment by psampathkumar** [comment]
No, this is also a relic of the past. We no longer even use the ranking formula.. We can probably replace it by a league ranking table and keep this in archive.
